modules/rodoo-enable-override.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+//
+// * nodes/pods/run_once_duration_override/run-once-duration-override-install.adoc
+
+:_content-type: PROCEDURE
+[id="rodoo-enable-override_{context}"]
+= Enabling the run-once duration override on a namespace
+
+To apply the run-once duration override from the {run-once-operator} to run-once pods, you must enable it on each applicable namespace.
+
+.Prerequisites
+
+* You have installed the {run-once-operator}.
+* You have access to the cluster with `cluster-admin` privileges.
++
+// TODO: are cluster-admin privileges required here to label? what if it's their own namespace?
+
+.Procedure
+
+. Log in to the OpenShift CLI.
+
+. Add the label to enable the run-once duration override to your namespace:
++
+[source,terminal]
+----
+$ oc label namespace <namespace> \ <1>
+    runoncedurationoverrides.admission.runoncedurationoverride.openshift.io/enabled=true
+----
+<1> Specify the namespace to enable the run-once duration override on.
+
+After you enable the run-once duration override on this namespace, future pods that are created in this namespace will have their `activeDeadlineSeconds` field set to the override from the {run-once-operator}. Existing pods in this namespace will also have their `activeDeadlineSeconds` value set if they are updated.
+
+// TODO: work on this ^ wording. Be clearer about what we mean by if they are updated ^
```

Comment:
@ingvagabund Can you clarify what you mean about that if a pod is updated that it will get its activeDeadlineSeconds value set? In what scenarios could it be updated (manually by a user? some other way?) that it would get that value set?

Reply:
In theory we want to mutate every pod. However, an admission plugin does not get a list of pods. Only a single pod. So we have to wait until a pod gets updated (e.g. its status changes) so we can mutate the field. In theory it should not take more than few seconds for all pod statuses to get updated.